android/app/buildConfigs.gradle

```diff
@@ -1,58 +1,117 @@
-// See https://github.com/luggit/react-native-config/blob/master/android/dotenv.gradle for reference
+buildscript {
```

Comment:
Type Safety, there is no need to use the ugly XCConfig format. Also usually an Android developer never heared about XCConfig files. I switched because it made the code easier from the perspective of Android. Also this is how it works in JavaScript code.

Reply:
Okay, convinced :+1: Please look into my other comments for a simpler buildConfigs.gradle file though:
* removing the firebase setup to a separate location
* moving duplicated camelizing and upper casing back to the js side